README.md

```diff
@@ -45,44 +45,44 @@ Run `python3 data-apy.py --help` for an overview.
 ### Examples:
 Query data for collision sensor on BAI_000646 for the last 3 days:

```

Comment:
Missed a couple of these...

Reply:
argh, good catch!

Reply:
@orlandobrent - should be taken care of now on master